ssz-rs/examples/another_container.rs

```diff
@@ -1,6 +1,5 @@
 use hex;
 use ssz_rs::prelude::*;
-use std::iter::FromIterator;
```

Comment:
nice catch, I've removed these on `main` and you can rebase to pull in those changes

Reply:
ah I see this is in a few places, sorry for extra rebase work and we will keep the others in this PR :)